### PR TITLE
Add CC and BCC function in sendEmail

### PR DESCRIPTION
**Pull request by uttampanara**
Changes  in sendEmail function CC and BCC parameters added

## Changelog

``` markdown
### Added
- CC and BCC function added
```

```
if ($ccTo = $message->getValue('cc')) {
    $mail->setCc($ccTo);
}
if ($bccTo = $message->getValue('bcc')) {
    $mail->setBcc($bccTo);
}
```
## Subject

Add CC and BCC parameters in sendEmail function

### CODE REVIEW

**Comment by OskarStark** [comment]
Please fill in the PR Template in the header @uttampanara

File: Consumer/SwiftMailerConsumer.php [hunk 1]
```diff
@@ -64,7 +64,15 @@ private function sendEmail(MessageInterface $message)
         if ($replyTo = $message->getValue('replyTo')) {
             $mail->setReplyTo($replyTo);
         }
-
+
```
Comment (OskarStark) [inline Consumer/SwiftMailerConsumer.php]: this line is wrong, please keep the old one

**Comment by OskarStark** [comment]
Please adjust the `SwiftMailerConsumerTest.php`

**Comment by uttampanara** [comment]
We need to give functionality for BCC and CC at core level. At least SwiftMailer default parameters.
And i have tested this one. It's works.

File: Consumer/SwiftMailerConsumer.php [hunk 2]
```diff
@@ -64,15 +64,18 @@ private function sendEmail(MessageInterface $message)
...
+        if ($ccTo = $message->getValue('cc')) {
+            $mail->setCc($ccTo);
+        }
+        if ($bccTo = $message->getValue('bcc')) {
+            $mail->setBcc($bccTo);
+        }
         if ($text = $message->getValue(array('message', 'text'))) {
             $mail->addPart($text, 'text/plain');
         }
-
```
Comment (OskarStark) [inline Consumer/SwiftMailerConsumer.php]: please keep the blank line

File: Consumer/SwiftMailerConsumer.php [hunk 3]
```diff
@@ -64,15 +64,18 @@ private function sendEmail(MessageInterface $message)
...
         if ($html = $message->getValue(array('message', 'html'))) {
             $mail->addPart($html, 'text/html');
         }
-
```
Comment (OskarStark) [inline Consumer/SwiftMailerConsumer.php]: please keep the blank line

**Comment by OskarStark** [comment]
- [ ] Please complete the first sentence of the PR
- [ ] if there are no related issues, please remove `Fixes` and `Closes`
- [ ] apply my comments
- [ ] use a good commit message

**Comment by OskarStark** [comment]
ping @uttampanara 

@Soullivaneuh stuck build @StyleCi

**Comment by uttampanara** [comment]
Add CC and BCC parameters in sendEmail function

**Comment by core23** [comment]
Any updates here @uttampanara ?

**Comment by uttampanara** [comment]
Nop

**Comment by greg0ire** [comment]
Closing then.

**Pull request closed without merging**